x/auth/keeper/querier_test.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/x/auth/exported"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+func TestQueryAccount(t *testing.T) {
+	input := SetupTestInput()
+
+	req := abci.RequestQuery{
+		Path: "",
+		Data: []byte{},
+	}
+
+	querier := NewQuerier(input.Ak)
+
+	bz, err := querier(input.Ctx, []string{"other"}, req)
+	require.Error(t, err)
+	require.Nil(t, bz)
+
+	req = abci.RequestQuery{
+		Path: fmt.Sprintf("custom/%s/%s", types.QuerierRoute, types.QueryAccount),
+		Data: []byte{},
+	}
+	res, err := queryAccount(input.Ctx, req, input.Ak)
+	require.Error(t, err)
```

Comment:
why did this behavior change? previous querier_test.go file had this has `require.NotNil`

Reply:
It hasn't. From [godoc](https://godoc.org/github.com/stretchr/testify/require#Error): 

> Error asserts that a function returned an error (i.e. not `nil`).

I think it's more clear to read the test this way

Reply:
oh yea, my mistake! Not sure why I was registering `Error` and `NotNil` as opposites..